sdk/eidolon_ai_sdk/builtins/logic_units/api_helper.py

```diff
@@ -15,7 +16,14 @@ async def get_content(url: str, headers=None, **kwargs):
     async with AsyncClient(timeout=Timeout(5.0, read=600.0)) as client:
         response = await client.get(**params, **kwargs)
         await AgentError.check(response)
-        return response.json()
+        content_type = response.headers.get('Content-Type', '')
+        if 'application/json' in content_type:
+            return response.json()
+        # Needs to include text/plain because some specs return text/plain for yaml like anything from github
+        elif 'application/x-yaml' in content_type or 'text/yaml' in content_type or 'text/plain' in content_type:
```

Comment:
@dbrewster so I needed to include text/plain here because that's what github returns on .yaml specs like this one: https://raw.githubusercontent.com/APIs-guru/openapi-directory/main/APIs/googleapis.com/calendar/v3/openapi.yaml

Not sure if this is the best way to handle all this though

Reply:
Let's not do this for yaml.  In GitHub you can request the openapi.json and it converts for you.

Reply:
@dbrewster what are the steps to do that?